standard-components/pom.xml

```diff
@@ -80,27 +90,27 @@
 		</dependency>
 		<dependency>
 			<groupId>org.eobjects.datacleaner</groupId>
-			<artifactId>DataCleaner-json</artifactId>
+			<artifactId>DataCleaner-sample-extension</artifactId>
```

Comment:
We don't want the sample-extension in here. It's only a coding example, not something to actually use!